[MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup

AnnaAPak opened the pull request:
### Description (*)
Adding AdminSelectAttributeSetOnEditProductPageActionGroup


### Resolved issues:
1. [x] resolves magento/magento2#31290: [MFTF] Adding AdminSelectAttributeSetOnEditProductPageActionGroup

AnnaAPak committed:
added AdminSelectAttributeSetActionGroup

AnnaAPak committed:
updated tests with new ActionGroup

AnnaAPak committed:
refactored

AnnaAPak committed:
added AdminSelectAttributeSetOnEditProductPageActionGroup

AnnaAPak commented:
@magento run Functional Tests B2B, Functional Tests EE

sidolov commented:
@magento create issue

AnnaAPak commented:
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

eduard13 commented on app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSelectAttributeSetOnEditProductPageActionGroup.xml:
```diff
@@ -0,0 +1,24 @@
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectAttributeSetOnEditProductPageActionGroup">
+        <annotations>
+            <description>Selects the specified value from the Attribute Set dropdown.
+                         The Edit Product Page should be opened prior to Action Group execution</description>
```
I would refactor this part. As for `The Edit Product Page should be opened prior to Action Group execution` sentence, I think that is extra. Usually, you are checking the ActionGroup implementation before using it.

```suggestion
            <description>Selects the specified value from the Attribute Set dropdown on the opened product edit page.</description>
```

eduard13 reviewed (changes requested):
Hi @AnnaAPak could you please check the minor comment that I left?
Thank you.

AnnaAPak committed:
refactored

AnnaAPak committed:
adjusting annotation
Merge branch 'ref-AdminChangeProductAttributeSetTes' of github.com:AnnaAPak/magento2 into ref-AdminChangeProductAttributeSetTes

AnnaAPak committed:
Update app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminSelectAttributeSetOnEditProductPageActionGroup.xml

Co-authored-by: Eduard Chitoraga <e.chitoraga@atwix.com>

AnnaAPak committed:
refactored

AnnaAPak commented:
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

AnnaAPak commented:
@eduard13  can you please take a look one more time? Thank you.

eduard13 commented:
@magento run all tests

engcom-Hotel commented:
@magento run Functional Tests B2B, Functional Tests CE

engcom-Hotel commented:
@magento run Functional Tests B2B, Functional Tests CE

engcom-Alfa commented:
QA not applicable
